dev/io.openliberty.jakarta.concurrency.3.0_fat_tck/fat/src/io/openliberty/jakarta/enterprise/concurrent/tck/ConcurrentTckLauncher.java

```diff
@@ -0,0 +1,124 @@
+ * location.
+ */
+@RunWith(FATRunner.class)
+public class ConcurrentTckLauncher {
+
+    final static Map<String, String> additionalProps = new HashMap<>();
+
+    @Server("ConcurrentTCKServer")
+    public static LibertyServer server;
+
+    @BeforeClass
+    public static void setUp() throws Exception {
+        //username and password for Arquillian to authenticate to restConnect
+        additionalProps.put("tck_username", "arquillian");
+        additionalProps.put("tck_password", "arquillianPassword");
+
+        //Keystore and Truststore for Arquillian to perform TLS handshake with Liberty server
+        additionalProps.put("javax.net.ssl.keyStore", server.getServerRoot() + "/resources/security/arquillian.p12");
+        additionalProps.put("javax.net.ssl.keyStorePassword", "arquillianPassword");
+        additionalProps.put("javax.net.ssl.keyStoreType", "pkcs12");
+        additionalProps.put("javax.net.ssl.trustStore", server.getServerRoot() + "/resources/security/arquillian.p12");
+        additionalProps.put("javax.net.ssl.trustStorePassword", "arquillianPassword");
+        additionalProps.put("javax.net.ssl.trustStoreType", "pkcs12");
+        //additionalProps.put("javax.net.debug", "ssl:handshake:verbose:keymanager:trustmanager");
+
+        //Logging properties for java.util.logging to use for mvn output
+        additionalProps.put("java.util.logging.config.file", server.getServerRoot() + "/resources/logging/logging.properties");
+
+        //username and password to set on quickStartSecurity
+        server.addEnvVar("tck_username", "arquillian");
+        server.addEnvVar("tck_password", "arquillianPassword");
+
+        //Keystore and Trustore for Liberty to perform TLS handshake with Liberty server
+        server.addEnvVar("tck_tls_store", server.getServerRoot() + "/resources/security/liberty.p12");
+        server.addEnvVar("tck_tls_password", "libertyPassword");
+
+        //Ports liberty should be using for testing
+        server.addEnvVar("tck_port", "" + server.getPort(PortType.WC_defaulthost));
+        server.addEnvVar("tck_port_secure", "" + server.getPort(PortType.WC_defaulthost_secure));
+
+        //Finally start the server
+        server.startServer();
+    }
+
+    @AfterClass
+    public static void tearDown() throws Exception {
+        server.stopServer(
+                          "WLTC0032W", //Transaction rollback warning.
+                          "WLTC0033W", //Transaction rollback warning.
+                          "CWWKS0901E" //Quickstart security
+        );
+    }
+
+    /**
+     * Run the TCK (controlled by autoFVT/publish/tckRunner/tck/*)
+     */
+    //@Test //TODO Enable once TCK is published on maven central
+    @AllowedFFDC // The tested exceptions cause FFDC so we have to allow for this.
+    public void launchConcurrentTCK() throws Exception {
+        String suiteXmlFile;
+        if (TestModeFilter.FRAMEWORK_TEST_MODE == Mode.TestMode.FULL) {
```

Comment:
I checked with Tom Evans, who confirmed that the build from which we collect official TCK results runs in FULL mode, so we should be fine with this split.

Reply:
Awesome, thanks for checking on that.